### PR TITLE
Fix issues in analysis module

### PR DESCRIPTION
**Pull request by CermakM**
- Do not raise on missing packages when building dep table 
- Fix typo in prediction: return s/winner_scores/scores

### CODE REVIEW

File: thoth/build_analysers/preprocessing.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def build_log_to_dependency_table(log: str) -> pd.DataFrame:
     df = pd.io.json.json_normalize(parse_log(log), record_path="result")
 
     if len(df) <= 0:
-        raise ValueError("No packages have been found in the log file.")
```
Comment (goern) [inline thoth/build_analysers/preprocessing.py:63]: isnt an exception better to handle in the calling code than an empty DF?

Comment (CermakM) [reply thoth/build_analysers/preprocessing.py:63]: I think not, I think that raising is unnecessary and it should be up to the user to handle the case when no packages have been found, which is indicated by an empty DataFrame. I think we should not treat it as a critical failure if there are no packages found by the parser (consider for example successful pipenv non-verbose build log, which AFAIK does not have any package names in it and it is perfectly valid.

**Review by goern — DISMISSED** [review]

**Review by goern — APPROVED** [review]

**Comment by CermakM** [comment]
Thanks!

**Comment by goern** [comment]
you could use 'regate' as a single word comment and zuul will take care of it

**Pull request merged**